tests/__fixtures__/Rules/lowercase_title/fixture.js

```diff
@@ -0,0 +1,143 @@
+const generator = require("../../../helpers/problemGenerator");
+const LowercaseTitle = require("../../../../lib/rules/lowercase_title");
+
+function generateProblem(location) {
+    return generator(LowercaseTitle, location, LowercaseTitle.meta.message, {
+        applyFix: jest.fn(),
+    });
+}
+
+function getValidTestData() {
+    return [
+        [
+            "Feature - Rule - Background - Scenario - Scenario Outline",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+    Scenario: a scenario
+    Scenario Outline: a scenario`,
+            [],
+        ],
+        [
+            "Feature - Rule - Scenario",
+            `Feature: a feature file
+  Rule: a rule
+    Scenario: a scenario`,
+            [],
+        ],
+    ];
+}
+
+function getInvalidTestData() {
```

Comment:
I suggest that you have some invalid test examples that have uppercase characters not in the first position.
Things like `Feature: a Feature File`
That will verify that the code works on the whole string.

Reply:
Okay i will correct that